paddle/fluid/pybind/eager.h

```diff
@@ -21,7 +21,8 @@ namespace paddle {
 namespace pybind {
 
 typedef struct {
-  PyObject_HEAD paddle::experimental::Tensor tensor;
+  PyObject_HEAD 
+  paddle::experimental::Tensor tensor;
```

Comment:
<img width="960" alt="image" src="https://user-images.githubusercontent.com/31559413/189038738-23fa66c0-10d1-4cad-b65b-2bf79c993e3f.png">
Include\object.h

Reply:
It's better to put PyObject_HEAD  at a single line.